Userland/Applications/PixelPaint/ImageMasking.cpp

```diff
@@ -437,8 +437,8 @@ double ColorWheelWidget::hue()
 void ColorWheelWidget::calc_hue(Gfx::IntPoint const& position)
 {
     auto center = Gfx::IntPoint(width() / 2, height() / 2);
+    auto angle = AK::to_degrees(AK::atan2(static_cast<float>(position.y() - center.y()), static_cast<float>(position.x() - center.x())));
```

Comment:
The commit message ends mid-sentence :)

Reply:
Sorry for that... it has already been late after work.